examples/cpp/logger_example/logger_w_sinks/logger_w_sinks_example.cpp

```diff
@@ -0,0 +1,72 @@
+/**
+ * Example of a logger that has externally defined sinks
+ * 
+ */
+
+#include "../logger_example_utils/logger_example_utils.h"
+
+using namespace daq;
+
+// (note on sinks)
```

Comment:
You have a few placeholder comments left here, you can just take them out if they're not necessary.